src/js/components/molecules/filter-category/filter-category.ts

```diff
@@ -23,13 +28,64 @@ export class FilterCategory extends Vue implements FilterCategoryProps {
      */
     $onPublic;
 
+    filterList: any;
```

Comment:
If you define the shape / interface of the filterList you can remove the // eslint-disable on line #52